src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -397,7 +408,7 @@ private void loadProcessingFilesConfigDeprecated(String componentName) {
             currentProcessingComponentTopicsDeprecated.iterator().forEachRemaining(node ->
                     processingFileInformation.updateFromTopic((Topic) node));
 
-            processingFiles.put(processingFileInformation);
+            processingFiles.putIfAbsent(processingFileInformation);
```

Comment:
I think we should update the last accessed time so we should still use `put` instead of `putIfAbsent`?

Reply:
In this case I don't want it here to be put again if it is already there. This method gets called when the configuration gets loaded. It will only be empty when we are booting up GG or we get restarted but for the rest we just want to put the file info if absent.

Since this values come from the config we don't want to update the lastAccessed time here.